cartographer/mapping/internal/optimization/optimization_problem_2d.cc

```diff
@@ -207,7 +208,7 @@ void OptimizationProblem2D::InsertTrajectoryNode(const NodeId& node_id,
 }
 
 void OptimizationProblem2D::TrimTrajectoryNode(const NodeId& node_id) {
-  imu_data_.Trim(node_data_, node_id);
+  empty_imu_data_.Trim(node_data_, node_id);
```

Comment:
We can keep this, it does nothing if it's empty. It's probably also good in case we ever use IMU again, so we don't forget about this part.